README.md

```diff
@@ -209,6 +209,19 @@ ACI can be cumbersome, so acbuild provides a context free mode. The
 when provided the current build context will be ignored and the change will be
 applied to the given ACI instead.
 
+## Trying out acbuild using Vagrant
```

Comment:
Can you actually move this up into the "Installation" section please? Want to make sure it's front and center :/